src/Search/Elastica/IndexManager.php

```diff
@@ -128,15 +128,25 @@ public function __construct(string $prefix, array $indexes = [], array $types =
         }
 
         try {
-            if (!$index->exists()) {
-                $settings['analysis'] = [
-                    'analyzer'   => $mapper->getCustomAnalyzers(),
-                    'normalizer' => $mapper->getCustomNormalizers(),
-                ];
+            $settings['analysis'] = [
+                'analyzer'   => $mapper->getCustomAnalyzers(),
+                'normalizer' => $mapper->getCustomAnalyzers(),
+            ];
 
-                $index->create($settings);
-            }
+            $index->create($settings);
         } catch (\Throwable $e) {
+            if (strpos($e->getMessage(), 'resource_already_exists_exception')
+                || strpos($e->getMessage(), 'already exists')
+            ) {
+                try {
+                    $client->connect();
+                    $index->setMapping($mapper->getMapping($qname));
```

Comment:
@albertgcsula old elastica is slightly different.  ref how the update process works and use that.  iirc, they are not the same